src/Libraries/Microsoft.Extensions.AI/ChatCompletion/FunctionInvokingChatClient.cs

```diff
@@ -458,46 +492,23 @@ private static bool CopyFunctionCalls(
     }
 
     /// <summary>Updates <paramref name="options"/> for the response.</summary>
-    /// <returns>true if the function calling loop should terminate; otherwise, false.</returns>
-    private static bool UpdateOptionsForMode(ContinueMode mode, ref ChatOptions options, string? chatThreadId)
+    private static void UpdateOptionsForMode(ref ChatOptions options, string? chatThreadId)
```

Comment:
The "Mode" in this name was about ContinueMode, which is now gone. Seems like this should be renamed.